### PR TITLE
Quickwit

### PR DESCRIPTION
**Pull request by fulmicoton**

### CODE REVIEW

File: src/termdict/sstable_termdict/sstable/vint.rs [hunk 1]
```diff
@@ -0,0 +1,66 @@
+const CONTINUE_BIT: u8 = 128u8;
```
Comment (fmassot) [inline src/termdict/sstable_termdict/sstable/vint.rs:1]: can move it to common with vint on u32

File: src/termdict/sstable_termdict/termdict.rs [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+        .expect("Creating a TermDictionaryBuilder in a Vec<u8> should never fail")
+        .finish()
+        .expect("Writing in a Vec<u8> should never fail");
+    FileSlice::from(term_dictionary_data)
+});
+
+/// The term dictionary contains all of the terms in
+/// `tantivy index` in a sorted manner.
+///
+/// The `Fst` crate is used to associate terms to their
```
Comment (fmassot) [inline src/termdict/sstable_termdict/termdict.rs:84]: typo: `fst`

File: src/termdict/sstable_termdict/sstable/mod.rs [hunk 3]
```diff
@@ -0,0 +1,359 @@
...
+    #[allow(dead_code)]
+    pub fn write(&mut self, key: &[u8], value: &TValueWriter::Value) -> io::Result<()> {
+        self.write_key(key);
+        self.write_value(value)?;
+        Ok(())
+    }
+
+    pub fn write_value(&mut self, value: &TValueWriter::Value) -> io::Result<()> {
+        self.delta_writer.write_value(value);
+        self.num_terms += 1u64;
```
[the remark this answers is not in the document]
Comment (fulmicoton) [reply src/termdict/sstable_termdict/sstable/mod.rs:174]: No. I usually tell it to the reader, not the compiler.

File: src/termdict/sstable_termdict/sstable/vint.rs (hunk 1 above)
Comment (fulmicoton) [reply src/termdict/sstable_termdict/sstable/vint.rs:1]: Yeah I created this PR by mistake... I just wanted to use github to see the diff.

**Pull request closed without merging**